sormas-backend/src/main/java/de/symeda/sormas/backend/user/KeycloakService.java

```diff
@@ -269,17 +258,8 @@ private String createUser(Keycloak keycloak, User user) {
         }
 
         String[] pathSegments = response.getLocation().getPath().split("/");
-        String userId = pathSegments[pathSegments.length - 1];
-
-        try {
-            ensureRoles(keycloak, userId, user.getUserRoles());
-        } catch (Exception e) {
-            logger.warn("Cannot set the user roles property, will remove the user");
-            keycloak.realm(REALM_NAME).users().delete(userId);
-            throw e;
-        }
 
-        return userId;
+        return pathSegments[pathSegments.length - 1];
```

Comment:
Maybe add java doc or a comment what the return value looks like. Might be confusing for people to understand what exactly is returned and why we need the `-1`.